ChangeLog

```diff
@@ -1,6 +1,9 @@
 CHANGES
 =======
 
+* Add note in README about Python 2 compatibility
+* Make uninstallable on Python < 3.3
+
```

Comment:
This will be overwritten on the deploy to your squashed commit message

Reply:
I didn't do that. o__O